### ISSUE TITLE
Anthracycline exposure

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10240]
**Preferred term label:**
Anthracycline exposure

**Synonyms**
History of treatment with an anthracycline chemotherapy agent

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C5437452

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dilated cardiomyopathy caused by anthracycline

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000019
- label: Anthracycline exposure
- parents: HP:4000112
- pmid: 
- def: History of treatment with an anthracycline chemotherapy agent.
- comment: 
- orcid: orcid:0000-0001-8481-3559
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10240
- synonyms:
